Bottom: 3dbb13d321649e0f5a5d966ac5a96ee9d57aafae
Top:    616eee86fe606ee581b09ed5bf85d9eac61ddebf
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-17 16:38:19 +0800

Refresh of vhost_net-batch-update-used

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 058efe9..18c013d 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -509,8 +509,8 @@ static int get_tx_bufs(struct vhost_net *net,
 	struct vhost_virtqueue *vq = &nvq->vq;
 	int ret;
 
-	ret = vhost_net_tx_get_vq_desc(net, vq, vq->iov + nvq->done_idx,
-				       ARRAY_SIZE(vq->iov) - nvq->done_idx,
+	ret = vhost_net_tx_get_vq_desc(net, vq, vq->iov,
+				       ARRAY_SIZE(vq->iov),
 				       out, in, busyloop_intr);
 	if (ret)
 		goto err;
